src/Bicep.Core.IntegrationTests/ImportTests.cs

```diff
@@ -159,7 +159,7 @@ public void Using_import_statements_frees_up_the_namespace_symbol()
         public void You_can_swap_imported_namespaces_if_you_really_really_want_to()
         {
             var result = CompilationHelper.Compile(ServicesWithImports, @"
-import 'az@1.0.0' as sys
+import 'br/public:az@1.0.0' as sys
 import 'sys@1.0.0' as az
```

Comment:
What does `'sys@1.0.0'` mean here? Should we block this syntax?

Reply:
I don't really understand the question, maybe we can discuss more offline?

Reply:
Sure! I'd forgotten we only currently support this versioned import syntax for built-ins - I thought we also supported `import sys`.

Long-term, I think it's worth us thinking about what the import looks like for "built-ins" from a user experience perspective. For a user, it feels inaccurate to fix it to `@1.0.0` - it implies that there won't ever be changes to sys between different versions of Bicep (which is incorrect).

Regardless, I think we could consider this out of scope for the types-via-registry work.

Reply:
I did a write-up for this here: #12202

Reply:
Thank you I appreciate you doing that!❤️